modules/layers/src/arc-layer/arc-layer.ts

```diff
@@ -147,6 +147,10 @@ export default class ArcLayer<DataT = any, ExtraPropsT extends {} = {}> extends
     model?: Model;
   };
 
+  getBounds(): [number[], number[]] | null {
+    return super.getBounds(['instanceSourcePositions', 'instanceTargetPositions']);
```

Comment:
Nit: I find `super()` to be really tricky in practical use, it surprises me often even in code I wrote myself. Could we make another function that we can call instead of `super()`? Like `return this.getBoundsForAttributes(...)`